### PR TITLE
Fix docstring format

### PR DESCRIPTION
**Pull request by tv3141**
Related to #3065

### CODE REVIEW

**Comment by tv3141** [comment]
An unrelated test is failing in Travis.

**Comment by tv3141** [comment]
Close and reopen to rerun CI.

File: lib/iris/experimental/stratify.py [hunk 1]
```diff
@@ -82,31 +82,33 @@ def relevel(cube, src_levels, tgt_levels, axis=None, interpolator=None):
     that are generally monotonic in the direction of interpolation, such as
     height/pressure or salinity/depth.
 
-    Parameters
```
Comment (pelson) [inline lib/iris/experimental/stratify.py]: This change is going in the wrong direction IMO - the numpydoc convention is the desired approach long-term. That said, it requires quite extensive change throughout the codebase, and improved tooling to achieve, so I'm completely OK with fixing this here and now if necessary.

@tv3141 - did you take a look at what happens if we install napolean / numpydoc extensions - do the RST warnings persist?

Comment (tv3141) [reply lib/iris/experimental/stratify.py]: `stratify.py` (and the scipy code in [lib/iris/analysis/_scipy_interpolate.py](https://github.com/SciTools/iris/blob/2ac14cb01e771aa2fdc04ebbbdf9a2aec60aa558/lib/iris/analysis/_scipy_interpolate.py)) are the only modules that follow the numpydoc style by using:
```
Parameters
----------
```
For consistency I stuck to the [devguide](https://scitools.org.uk/iris/docs/latest/developers_guide/documenting/docstrings.html#sample-multi-line-docstring).

I will try adding the numpydoc extension.

File: lib/iris/experimental/stratify.py [hunk 2]
```diff
@@ -82,31 +82,33 @@ def relevel(cube, src_levels, tgt_levels, axis=None, interpolator=None):
...
-    ----------
-    cube : :class:`~iris.cube.Cube`
+    Args:
+
+    * cube (`iris.cube.Cube`):
```
Comment (dkillick) [inline lib/iris/experimental/stratify.py]: @tv3141 why have the intersphinx references been removed throughout this change? Are the references not desired, or am I missing something?

Comment (tv3141) [reply lib/iris/experimental/stratify.py]: @dkillick By changing the style of the docstring a line became too long and caused a Stickler error. I could not find a way to break a bullet point without violating rst syntax, and thus shortened the line by removing the references.
I will look into using `numpydoc`, it should fix the original issue without any other changes.

File: lib/iris/experimental/stratify.py (hunk 1 above)
Comment (tv3141) [reply lib/iris/experimental/stratify.py]: `numpydoc`fixes this warning but creates ~1500 identical new ones. 
https://github.com/tv3141/iris/tree/use_sphinx_numpydoc
```
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.add_aux_coord'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.add_aux_factory'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.add_cell_measure'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.add_cell_method'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.add_dim_coord'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.aggregated_by'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.aux_factory'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.cell_measure'
/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/scitools_iris-2.2.0.dev0-py2.7.egg/iris/cube.py:docstring of iris.cube.Cube:118: WARNING: toctree references unknown document u'iris/iris/iris.cube.Cube.cell_measure_dims'
```

**Pull request closed without merging**